beacon_node/src/cli.rs

```diff
@@ -316,19 +320,22 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 present in the configuration, the quotas used for the inbound rate limiter will be \
                 used."
             )
-            .min_values(0)
-            .hidden(true)
+            .default_missing_value("true")
+            .require_equals(false)
+            // TODO this is dangerous without requires_equals = true
+            .num_args(0..1)
+            .hide(true)
```

Comment:
see comment regarding `--latency-measurement-service` flag why this configuration feels "bad".

I suggest introducing a new flag `--enable-self-limiter` with `ArgAction::SetTrue` and changing the `--self-limiter` flag to `ArgAction::Append`.

if the `self-limiter` flag is provided with value(s), self-limiter will be enabled with the provided value(s).  if `--enable-self-limiter` flag is provided, the quotas for the inbound rate limiter will be used. (Also do the same for the `inbound-rate-limiter` flag)

Reply:
Yep, lets do this to make it more compatible with the new clap